src/dsp_tools/utils/xml_upload.py

```diff
@@ -251,7 +251,7 @@ def _stash_circular_references(
                                 stashed_resptr_props[res][link_prop].append(str(value.value))
                         link_prop.values.remove(value)
             else:
-                logger.exception("ERROR in remove_circular_references(): link_prop.valtype is neither text nor resptr.")
+                logger.error("ERROR in remove_circular_references(): link_prop.valtype is neither text nor resptr.")
```

Comment:
logging.exception() is a shorthand for logging.error(exc_info=True), which should only be called inside an except block.